docs-web/configuration/policy-resource.md

```diff
@@ -335,6 +341,77 @@ policies:
 ```
 In this example the Ingress Controller will use the configuration from the first policy reference `ingress-mtls-policy-one`, and ignores `ingress-mtls-policy-two`.
 
+### EgressMTLS
+
+The EgressMTLS policy configures upstreams authentication and certificate verification.
+
+For example, the following policy will use `egress-mtls-secret` to authenticate with the upstream application and `egress-trusted-ca-secret` to verify the certificate of the application:
+```yaml
+egressMTLS:
+  tlsSecret: egress-mtls-secret
+  trustedCertSecret: egress-trusted-ca-secret
+  verifyServer: on
+  verifyDepth: 2
+```
+
+> Note: The feature is implemented using the NGINX [ngx_http_proxy_module](https://nginx.org/en/docs/http/ngx_http_proxy_module.html).
+
+```eval_rst
+.. list-table::
+   :header-rows: 1
+
+   * - Field
+     - Description
+     - Type
+     - Required
+   * - ``tlsSecret``
+     - The name of the Kubernetes secret that stores the TLS certificate and key. It must be in the same namespace as the Policy resource. The certificate must be stored in the secret under the key ``tls.crt`` and ``tls.key``, otherwise the secret will be rejected as invalid.
+     - ``string``
+     - No
+   * - ``trustedCertSecret``
+     - The name of the Kubernetes secret that stores the CA certificate. It must be in the same namespace as the Policy resource. The certificate must be stored in the secret under the key ``ca.crt``, otherwise the secret will be rejected as invalid.
+     - ``string``
+     - No
+   * - ``verifyServer``
+     - Enables verification of the upstream HTTPS server certificate.
+     - ``bool`
+     - No
+   * - ``verifyDepth``
+     - Sets the verification depth in the client certificates chain. The default is ``1``.
+     - ``int``
+     - No
+   * - ``sessionReuse``
+     - Enables reuse of SSL sessions to the upstreams. The default is ``true``.
+     - ``bool`
+     - No
+   * - ``serverName``
+     - Enables passing of the server name through ``Server Name Indication`` extension.
+     - ``bool`
+     - No
+   * - ``sslName``
+     - Allows overriding the server name used to verify the certificate of the upstream HTTPS server.
+     - ``string`
+     - No
+   * - ``ciphers``
+     - Specifies the enabled ciphers for requests to an upstream HTTPS server. The default is ``DEFAULT``.
```

Comment:
we need to put a note here that explains that this and the field below aren't validated. How about this?

Note: the value is not validated by the Ingress Controller. As a result, NGINX can fail to reload the configuration. To ensure that the configuration for a VirtualServer/VirtualServerRoute that references the policy was successfully applied, check its [status](https://deploy-preview-1186--nginx-kubernetes-ingress.netlify.app/nginx-ingress-controller/configuration/global-configuration/reporting-resources-status/#virtualserver-and-virtualserverroute-resources). The validation will be added in the future releases.